### PR TITLE
chore(dashboard): add new e2e tests

### PR DESCRIPTION
**Pull request by suddjian**
### SUMMARY

Adding a few new tests as a follow-up

### TEST PLAN

run the tests...

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/dashboard/components/gridComponents/Chart.jsx [hunk 1]
```diff
@@ -297,7 +297,7 @@ export default class Chart extends React.Component {
         data-test="chart-grid-component"
         data-test-chart-id={id}
         data-test-viz-type={slice.viz_type}
-        data-test-chart-name={slice.slice_name}
+        data-test-chart-name={sliceName}
```
Comment (suddjian) [inline superset-frontend/src/dashboard/components/gridComponents/Chart.jsx:300]: this properly includes any name overrides used in the dashboard

File: superset-frontend/cypress-base/cypress/integration/dashboard/dashboard.helper.ts [hunk 2]
```diff
@@ -110,8 +119,8 @@ export function drag(selector: string, content: string | number | RegExp) {
       (typeof target === 'string' ? cy.get(target) : target)
         .trigger('dragover', { dataTransfer })
         .trigger('drop', { dataTransfer })
-        .trigger('dragend', { dataTransfer })
-        .trigger('mouseup', { which: 1 });
+        .trigger('dragend', { dataTransfer, force: true })
+        .trigger('mouseup', { which: 1, force: true });
```
Comment (suddjian) [inline superset-frontend/cypress-base/cypress/integration/dashboard/dashboard.helper.ts:123]: this handles cases where dragging something onto a drop target causes the drop target to become covered by new elements

**Comment by rusackas** [comment]
Is this PR still needed? Seems to have fallen by the wayside.

**Comment by amitmiran137** [comment]
Lets close and re-open if needed

**Pull request closed without merging**